SCSS/_varieables.scss

```diff
@@ -1 +1,4 @@
-$primary-color: pink;
+$primary-color: #DB720D;
```

Comment:
nazwy dla zmiennych nie muszą mieć -color bo i tak wiadomo przecież z składni że to są kolory - możemy to jednak zostawić tak tylko zwracam uwagę że lepszą nazwą mogło by być np. $primary-text - a to że jest to kolor widać co jest do niego przypisane czyli np. #DB720D